guacvisualizer.md

```diff
@@ -0,0 +1,179 @@
+---
+layout: page
+title: Set up GUAC Visualizer
+permalink: /guac-visualizer
+nav_order: 3
+---
+
+# Set up GUAC Visualizer
+
+The GUAC Visualizer is an experimental utility that can be used to interact with
+the GUAC services. It acts as a way to visualize the software supply chain graph,
+as well as a means to explore the supply chain and prototype policies.
+
+Since the GUAC Visaulizer is still in an early experimental stage, it is likely
+that there may be some unexpected behavior or usage problems. For a more robust
+use of GUAC, we recommend using the
+[GraphQL interface directly](https://github.com/guacsec/guac/blob/main/demo/GraphQL.md).
```

Comment:
Is the GraphQL interface going to be reflected somewhere in the docs if it's the primary mode of interaction?

Reply:
There is a graphql demo that Sarah is going to bring into the docs. She'll need to adjust the link once it is in.